src/deployment/obfuscate.md

```diff
@@ -116,6 +116,23 @@ use the following steps to make it human readable:
    For more information on the `symbolize` command,
    run `flutter symbolize -h`.
 
+## Reading an obfuscated name
+
+To understand a name created by an obfuscated app,
+use the following steps to make it human readable:
+
+1. At app build time, save the name obfuscation map
+   using `--extra-gen-snapshot-options=--save-obfuscation-map=/<your-directory>`.
```

Comment:
I'm curious why this line has a directory, yet the next command has a path. Does the tool generate it's own file(s) in the named target directory? Worth adding that clarification, I believe.

Reply:
My bad, it is (always) a path, fixed